jx_sqlite/setop_table.py

```diff
@@ -191,7 +191,7 @@ def place(parent_doc_details):
                     index_to_column[column_number] = nested_doc_details['index_to_column'][column_number] = ColumnMapping(
                         push_name=s.name,
                         push_column=si,
-                        push_child=relative_field(c.name, s.name),
+                        push_child=relative_field(c.names, s.name),
```

Comment:
the line will fail as it is now, which means you will be forced to revisit it later.